packages/menu/src/Menu.ts

```diff
@@ -34,6 +36,9 @@ export class Menu extends SpectrumElement {
         return [menuStyles];
     }
 
+    @property({ type: Boolean, reflect: true })
+    public selectable = false;
```

Comment:
is this just used for css right now? should there be new tests on this?

Reply:
CSS relative functionality in this change, so this is a pretty good test for it:

[![Open Selectable Menu under the current CI cycle](https://user-images.githubusercontent.com/1156657/107289884-ac359080-6a33-11eb-9bd8-567b915c4145.png)](https://circle-production-customer-artifacts.s3.amazonaws.com/picard/5bf34b122c07f700013ff527/6021b963d8a4a6132a424096-0-build/artifacts/test/visual/screenshots-current/ci/split-button-cta-field--open-menu-left__dark__large__ltr.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20210208T223222Z&X-Amz-SignedHeaders=host&X-Amz-Expires=60&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20210208%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=e85036540d54c5ff5a1666adcbab7be5d6da0acbb0d56ed83a84dbcbff9253aa)

as opposed to the previous delivery:

[![Open Selectable Menu under the `longpress` CI cycle](https://user-images.githubusercontent.com/1156657/107289962-ca9b8c00-6a33-11eb-8176-314b697e811a.png)](https://circle-production-customer-artifacts.s3.amazonaws.com/picard/5bf34b122c07f700013ff527/6021b940fa8b8630b0d334c9-0-build/artifacts/test/visual/screenshots-current/ci/split-button--cta-open__dark__large__ltr.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20210208T223325Z&X-Amz-SignedHeaders=host&X-Amz-Expires=60&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20210208%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=9b3609fedeba76f71a1ee22fd12e50c2eeea31654d7d861090cbb1bc97063015)

Notice how the "checkmark" was previously tucked within the text width of the other options.